api-server/api-server-common/src/storage/storage_api/mod.rs

```diff
@@ -829,6 +847,13 @@ pub trait ApiServerStorageWrite: ApiServerStorageRead {
         block_height: BlockHeight,
     ) -> Result<(), ApiServerStorageError>;
 
+    async fn set_token_transactions_at_height(
```

Comment:
Btw, this function also deserves a comment, because when called for an existing height, some `set_xxx_at_height` functions update the data and some keep the old one. So, the comment should mention what this particular function will do in this situation.